packages/usa-form/src/usa-form.stories.js

```diff
@@ -40,3 +47,10 @@ SignInMultiple.args = MultipleContent;
 
 export const SignInMultipleSpanish = SignInMultipleTemplate.bind({});
 SignInMultipleSpanish.args = EsMultipleContent;
+
+export const DisabledFormElements = CollectionTemplate.bind({});
+DisabledFormElements.argTypes = {
+  disabled_state: {
+    table: { disable: false },
+  },
+};
```

Comment:
We should follow the test pattern convention to keep these consistent
https://github.com/uswds/uswds/tree/develop/packages/usa-card/src/test/test-patterns

Reply:
I moved the twig and stories.js file into a test-patterns directory in [e301a4c](https://github.com/uswds/uswds/pull/5167/commits/e301a4c79279dabc90477192e93eda65b2b0f2d6). Let me know if you meant something different.

Reply:
Thanks for updating